SingularityUI/app/components/common/table/UITable.jsx

```diff
@@ -0,0 +1,325 @@
+import React, { Component, PropTypes } from 'react';
+import Waypoint from 'react-waypoint';
+import classNames from 'classnames';
+import _ from 'underscore';
```

Comment:
Kevin is correct but i think this is OK -- i'd eventually like to remove all of the globally scoped modules from the webpack config